nginx/dev/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM nginx:latest
+COPY ./nginx.conf /etc/nginx/nginx.conf
```

Comment:
I'm wondering if it's not easier to just mount the config at runtime to save ourselves building an image?